worker/worker.py

```diff
@@ -91,7 +92,14 @@ def _should_run(self):
         with self._runs_lock:
             if self._runs:
                 return True
-        return False 
+        return False
```

Comment:
Sorry about this, my editor automatically strips whitespace from the ends of lines.